integration-tests/tests/cases/mod.rs

```diff
@@ -257,22 +249,19 @@ async fn test_triple_persistence() -> anyhow::Result<()> {
 
 #[test(tokio::test)]
 async fn test_presignature_persistence() -> anyhow::Result<()> {
-    let docker_client = DockerClient::default();
-    let docker_network = "test-presignature-persistence";
-    docker_client.create_network(docker_network).await?;
-    let redis = containers::Redis::run(&docker_client, docker_network).await;
-    let redis_url = Url::parse(redis.internal_address.as_str())?;
-    let redis_cfg = deadpool_redis::Config::from_url(redis_url);
-    let redis_pool = redis_cfg.create_pool(Some(Runtime::Tokio1)).unwrap();
-    let presignature_storage = storage::presignature_storage::init(
-        &redis_pool,
-        &AccountId::from_str("test.near").unwrap(),
-    );
+    let spawner = ClusterSpawner::default()
```

Comment:
What is the benefit of spawning a cluster here?

Reply:
ClusterSpawner doesn't actually spawn a cluster here. It's used as a way to configure the env. In this case, it initializes the network for Redis. So wherever you `ClusterSpawner::default()`, it is configuration, not spawning vs `cluster::spawn`

Reply:
the spawner itself is the configuration though. It's only a couple places where we use it like this, so not worth it to clean it up right now